main.go

```diff
@@ -16,30 +16,39 @@ var rootCmd = &cobra.Command{
 	Short: "Secure Shell and Application Access Server",
 	Long:  "Mobingi Secure Shell and Application Access Server.",
 	Run: func(cmd *cobra.Command, args []string) {
-		log.Println("hello")
+		log.Println("hello!")
+		env := GetCliStringFlag(cmd, "env")
+		syslogging = GetCliStringFlag(cmd, "syslog")
+		if syslogging == "enable" {
+			logger, err = syslog.New(syslog.LOG_NOTICE|syslog.LOG_USER, "sesha3")
+			if err != nil {
+				panic(err)
+			}
+			log.SetOutput(logger)
+		}
 		_, err := os.Stat("./certs/")
 
 		if err == nil {
-			fmt.Println("./certs detected.")
+			log.Println("./certs detected.")
 		} else {
-			fmt.Println("./certs not detected. mkdir ./certs")
+			log.Println("./certs not detected. mkdir ./certs")
 			os.Mkdir("./certs", 0700)
 		}
 
-		env := GetCliStringFlag(cmd, "env")
 		domain = GetCliStringFlag(cmd, "domain")
 		region = GetCliStringFlag(cmd, "aws-region")
 		ec2id = GetCliStringFlag(cmd, "ec2-id")
 		credprof = GetCliStringFlag(cmd, "cred-profile")
 		err = awsports.Download(env, region, credprof)
-		fmt.Println(err)
+		log.Println(err)
 		serve(cmd)
 	},
 }
 
 func init() {
 	rootCmd.Flags().SortFlags = false
 	rootCmd.PersistentFlags().String("env", "dev", "values: dev, test, prod")
+	rootCmd.PersistentFlags().String("syslog", "disable", "values: enable, disable")
```

Comment:
@biosugar0 Try using bool instead of string.

Example:

```
var enabled bool
...
rootCmd.PersistentFlags().BoolVar(&enabled, "syslog", false, "set log output to syslog")
...
if enabled {
    // flag is enabled
}
```